public/app/plugins/datasource/cloudwatch/components/LogsCheatSheet.tsx

```diff
@@ -229,8 +229,15 @@ export default class LogsCheatSheet extends PureComponent<
       <div
         className="cheat-sheet-item__example"
         key={expr}
-        onClick={(e) =>
-          this.onClickExample({ refId: 'A', expression: expr, queryMode: 'Logs', region: 'default', id: 'A' })
+        onClick={() =>
+          this.onClickExample({
+            refId: this.props.refId,
```

Comment:
here you could do `this.props.query.refId ?? 'A'`

Reply:
Perfect, fixed.